entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+export PATH=${PATH}:~/.local/bin/
+
+cat << EOF > /opt/.env
```

Comment:
This is not needed, pydantic BaseSettings already will take needed variables from environment passed to container. `.env` file can just overwrite them.

Reply:
Got it, will remove spare code